awesomplete.js

```diff
@@ -301,9 +309,7 @@ function $$(expr, con) {
 	return slice.call((con || document).querySelectorAll(expr));
 }
 
-$.create = function(tag, o) {
-	var element = document.createElement(tag);
-
+$.set = function(element, o) {
```

Comment:
The main idea is that there are a lot of very useful things to have on any project, but usually no-one wants to take care of it. The solution is to automate, so it's ideally only done once.

I believe a lot of things can be improved. Some of them:

The package.json is still at "0.0.0" and no release on npm, and the 1.0 release was tagged on github a long time ago. Those need to be in sync. And the best way to have them in sync is to automate build/release process.

The version number. When someone opens a bug, it's very important to know the version they are referring to. Now it's like there is stable 1.0 and just latest. New version tag/release/whatever can be done with 1 command.

The changelog. It is always good to know about new features or which bugs are fixed in new version. I like how changelog generation is automated in Angular 2. They require commit messages in special format, so that it can be easily parsed. See https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit Do you think this might work for Awesomplete too?

I can start by replacing Makefile with gulp, so we'll have a base for other gulp tasks.